source/includes/kubernetes/_k8_ingresses.md

```diff
@@ -0,0 +1,158 @@
+## Ingresses
+
+<!-------------------- LIST INGRESSES -------------------->
+
+#### List ingresses
+
+```shell
+curl -X GET \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/a_service/an_environment/ingresses"
+```
+
+> The above command returns a JSON structured like this:
+
+```json
+{
+  "data": [
+    {
+      "id": "cloudmc/cmc-stg",
+      "endpoint": "http://cloudmc-stg.cloudops-devteam.com",
```

Comment:
Even though this endpoint is behind VPN but I am not sure about exposing valid endpoint in public docs. Should we update it with something like `http://cloudmc_staging_endpoint.com`? WYT?

Reply:
Yeah, good point. A pseudo endpoint would be a good idea.

Reply:
Changed all instances of real endpoint to `http://example.endpoint.com` and `http://fake.endpoint.com`, respectively, in [cc49492](https://github.com/cloudops/cloudmc-api-docs/pull/183/commits/cc49492a5299cdeab0cae44baabd50a59d50bcb9)